charts/fybrik/templates/katalog-connector/katalog-connector-rbac.yaml

```diff
@@ -5,18 +5,19 @@ apiVersion: rbac.authorization.k8s.io/v1
 {{- if .Values.clusterScoped }}
 kind: ClusterRoleBinding
 metadata:
-  name:  katalog-connector-viewer
+  name:  fybrik-katalog-connector-crb
```

Comment:
minor comment: In other places I see `{{ template "fybrik.fullname" . }}` is used, for example: https://github.com/fybrik/fybrik/blob/master/charts/fybrik/templates/fybrik-blueprints-rb.yaml#L5

Reply:
correct, Kritika from the BU works to replace all names to be with prefix "fybrik-" only,  (operator doesn't support templates. But meantime, no problem I'll add this here too.